airbyte-cdk/python/airbyte_cdk/sources/declarative/incremental/datetime_based_cursor.py

```diff
@@ -179,7 +179,7 @@ def _calculate_earliest_possible_value(self, end_datetime: datetime.datetime) ->
         cursor_datetime = self._calculate_cursor_datetime_from_state(self.get_stream_state())
         return max(earliest_possible_start_datetime, cursor_datetime) - lookback_delta
 
-    def _select_best_end_datetime(self) -> datetime.datetime:
+    def select_best_end_datetime(self) -> datetime.datetime:
```

Comment:
nit:  can you add a docstring explaining what "best end datetime" means from a user of the class since this is now a public method?